packages/client/hmi-client/src/components/widgets/tera-import-github-file.vue

```diff
@@ -188,18 +188,22 @@ import { getModeForPath } from 'ace-builds/src-noconflict/ext-modelist';
 import Checkbox from 'primevue/checkbox';
 import Dropdown from 'primevue/dropdown';
 import Breadcrumb from 'primevue/breadcrumb';
-import { createNewDatasetFromGithubFile } from '@/services/dataset';
-import { createNewArtifactFromGithubFile } from '@/services/artifact';
 import { extractPDF } from '@/services/knowledge';
 import useAuthStore from '@/stores/auth';
-import { uploadCodeToProjectFromGithub } from '@/services/code';
+import { useProjects } from '@/composables/project';
 
 const props = defineProps<{
 	urlString: string;
 	showImportButton: boolean;
 	project?: IProject;
 }>();
 
+const {
+	uploadCodeToProjectFromGithub,
+	createNewArtifactFromGithubFile,
+	createNewDatasetFromGithubFile
+} = useProjects();
```

Comment:
Those should not be in `useProject`, but in `useDataset`, `useArtifact`, `useCode` etc.